frontend/src/modules/Metadata_Forms/components/MetadataFormAttachedEntities.js

```diff
@@ -0,0 +1,350 @@
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Card,
+  CardContent,
+  CardHeader,
+  Divider,
+  Grid,
+  Typography
+} from '@mui/material';
+import {
+  deleteMetadataFormVersion,
+  getAttachedMetadataForm,
+  listAttachedMetadataForms,
+  listMetadataFormVersions
+} from '../services';
+import { SET_ERROR } from '../../../globalErrors';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useClient } from '../../../services';
```

Comment:
We should be using global imports instead of relative imports for services and globalErrors